tests/components/powerwall/test_sensor.py

```diff
@@ -27,7 +27,8 @@
         identifiers={("powerwall", "Wom Energy_60Hz_240V_s_IEEE1547a_2014_13.5")},
         connections=set(),
     )
-    assert reg_device.model == "PowerWall 2"
+    assert reg_device.model == "PowerWall 2 (hec)"
+    assert reg_device.sw_version == "1.45.1"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5281369)